recipes/ptplot/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "ptplot" %}
+{% set version = "0.3.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ptplot-{{ version }}.tar.gz
+  sha256: 4d23b74bae59a217e37001fbcb3ef9a99b640e5c8dce504847f82697e46c149a
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
+    - bokeh >=2.3.3
+    - nodejs >=14.17.0
+    - numpy >=1.19.5
+    - pandas >=1.2.0
+    - patsy >=0.5.1
+    - python >=3.7
+    - svgpathtools >=1.4.1
```

```suggestion
    - bokeh >=2.3.3,<3
    - nodejs >=14.17.0,<15
    - numpy >=1.19.5,<2
    - pandas >=1.2.0,<2
    - patsy >=0.5.1,<0.6.0
    - python >=3.7
    - svgpathtools >=1.4.1,<2
```

I believe we usually add an upper bound to most dependencies, assuming that they use semantic versioning.

---

Oh, ok sounds good. I committed the suggestion, thanks @saraedum!